knative-operator/deploy/resources/knativekafka/controller/eventing-kafka-controller.yaml

```diff
@@ -875,7 +875,6 @@ rules:
       - "*"
     resources:
       - configmaps
-      - services
```

Comment:
We have a PR on the midstream for adding this.

We set it on purpose here: 
https://github.com/openshift-knative/serverless-operator/pull/1535/commits/00ff78a6f5ce1a266fbf6a2d0b7219b3af03b702

Yes, the `lint` complains about it ... but that's OK